ros2param/ros2param/api/__init__.py

```diff
@@ -91,7 +91,7 @@ def get_parameter_value(*, string_value):
             value.string_value = string_value
     else:
         value.type = ParameterType.PARAMETER_STRING
-        value.string_value = yaml_value
+        value.string_value = yaml_value if yaml_value is not None else string_value
```

Comment:
if the string_value is empty or space, yaml_value becomes None, that is the cause of the exception. if it cannot convert it to yaml_value, which becomes None, store it as string value as it is.